dist/pivot.js

```diff
@@ -925,6 +926,7 @@
         localeStrings: locales.en.localeStrings
       };
       opts = $.extend(defaults, opts);
+      opts.aggregator = opts.aggregators[opts.aggregatorName](opts.vals);
```

Comment:
The API for `pivot()` assumes that the aggregator is given by the `aggregator` parameter, not looked up in an `aggregators` list, so this is a major API change, which I am not really in favour of.